### PR TITLE
6281 - post deploy health check smoketest

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: cypress-smoketests/integration/public/health-check.spec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+describe('Health check', () => {
+  it("should retrieve the status of the application's critial services", () => {
```
Comment (rachaelparris) [inline cypress-smoketests/integration/public/health-check.spec.js]: Nit: typo `critical`

File: cypress-smoketests/integration/public/health-check.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    cy.request({
+      followRedirect: true,
+      method: 'GET',
+      url: 'https://public-api.dev.ustc-case-mgmt.flexion.us/public-api/health',
```
Comment (rachaelparris) [inline cypress-smoketests/integration/public/health-check.spec.js]: Should this URL be dynamic based on the environment it's being run on?

File: cypress-smoketests/integration/public/health-check.spec.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }).should(response => {
+      console.log(response.body);
```
Comment (rachaelparris) [inline cypress-smoketests/integration/public/health-check.spec.js]: Can this be removed?

**Review by kkoskelin — APPROVED** [review]

File: cypress-smoketests/integration/public/health-check.spec.js (hunk 2 above)
Comment (rachelschneiderman) [reply cypress-smoketests/integration/public/health-check.spec.js]: Yes! Thank you!

**Pull request merged**